pkgs/modules/languageServers/dotreplit-lsp/default.nix

```diff
@@ -0,0 +1,59 @@
+{ pkgs, config, ... }:
+with pkgs.lib;
+let
+  cfg = config.languageServers.dotreplit-lsp;
+  # use changes from https://github.com/tamasfe/taplo/pull/510
```

Comment:
btw @cdmistman your stuff's been merged.

Reply:
yup! was able to get it merged after pinging. still hasn't been a release yet though so once that's done we can go back to relying on their standard releases :)